Patch status: limbo

Not cdefs header with musl.

When this issue comes up people are often told to use musl-compat or
bsd-compat-headers, and there seem to be generally little activity.

Patch taken from Void, there more are cdefs but on unused code paths.

https://crbug.com/1380656 (musl tracking bug)
--- a/src/3rdparty/chromium/third_party/crashpad/crashpad/compat/linux/sys/ptrace.h
+++ b/src/3rdparty/chromium/third_party/crashpad/crashpad/compat/linux/sys/ptrace.h
@@ -18,6 +18,4 @@
 #include_next <sys/ptrace.h>
 
-#include <sys/cdefs.h>
-
 // https://sourceware.org/bugzilla/show_bug.cgi?id=22433
 #if !defined(PTRACE_GET_THREAD_AREA) && !defined(PT_GET_THREAD_AREA) && \
--- a/src/3rdparty/chromium/third_party/libsync/src/include/sync/sync.h
+++ b/src/3rdparty/chromium/third_party/libsync/src/include/sync/sync.h
@@ -20,10 +20,11 @@
 #define __SYS_CORE_SYNC_H
 
-#include <sys/cdefs.h>
 #include <stdint.h>
 
 #include <linux/types.h>
 
-__BEGIN_DECLS
+#ifdef __cplusplus
+extern "C" {
+#endif
 
 struct sync_legacy_merge_data {
@@ -159,5 +160,7 @@
 void sync_fence_info_free(struct sync_fence_info_data *info);
 
-__END_DECLS
+#ifdef __cplusplus
+}
+#endif
 
 #endif /* __SYS_CORE_SYNC_H */
